docs/FAQ.md

```diff
@@ -49,8 +49,17 @@
 
    The workflow config file also supports setting base_images for workflows that may require special base images to run as well as an architecture field that will support configuring the CPU architecture for openshift to run on once non-amd64 architectures are supported by the ci-chat-bot (https://github.com/openshift/ci-chat-bot/blob/master/prow.go#L190).
 
+## workflow_test
+1. **What is `workflow_test`?**
 
-2. **How can I make a workflow runnable by the `workflow-launch` command**
+   Lorem ipsu,
```

Comment:
Appreciate the reference, but probably not very helpful for answering the question. ;)